### PR TITLE
Update Bevy version to 0.8

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Spotted in #6.

### CODE REVIEW

**Comment by mockersf** [comment]
@alice-i-cecile could you update to 0.8?

File: Cargo.toml [hunk 1]
```diff
@@ -5,4 +5,4 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-bevy = "0.6"
+bevy = "0.7"
```
Comment (alice-i-cecile) [inline Cargo.toml:8]:
```suggestion
bevy = "0.8"
```

**Comment by mockersf** [comment]
It needs a little more changes than just the version update, you can find the migration guide here: https://bevyengine.org/learn/book/migration-guides/0.7-0.8/

**Comment by alice-i-cecile** [comment]
Yep, working on that now :)

**Review by mockersf — APPROVED** [review]

**Pull request merged**